allure-generator/src/main/javascript/plugins/testcase-severity/SeverityView.hbs

```diff
@@ -0,0 +1,3 @@
+{{#if severity}}
+    <span>Severity: {{severity}}</span>
+{{/if}}
```

Comment:
Do you like to have so many redundant files? Instead of this, you can do inline template


```js
import {escapeExpression} from 'handlebars/runtime';

class SeverityView extends View {
   //..skipped

   template(data) {
     return severity ? `Severity ${escapeExpression(data.severity)}`;
   }
}

```